test/minikube/minikube_external_access_test.go

```diff
@@ -354,12 +376,15 @@ func TestKubernetesIngress(t *testing.T) {
 		}
 	}
 
-	// this requires CDriver to support Server Name Indication (SNI); there is
-	// no straightforward way to make this logic conditional because in internal
-	// CI we use the nuosql from the package installation but in CircleCI, we
-	// use the client package
-	// TODO: Change the version once DB-34466 is fixed
-	testlib.RunOnNuoDBVersionCondition(t, ">=5.1", func(version *semver.Version) {
+	// this requires CDriver to support Server Name Indication (SNI)
+	t.Run("sqlIngressConnectivity", func(t *testing.T) {
+		constraint, err := semver.NewConstraint(">=5.0.4")
```

Comment:
Nit: I see that DB-34502 is fixed in 5.0. Should we make the version constraint ">=5.0"?